sensing/gnss_poser/include/gnss_poser/gnss_poser_core.hpp

```diff
@@ -90,6 +90,7 @@ class GNSSPoser : public rclcpp::Node
   std::string gnss_base_frame_;
   std::string map_frame_;
 
+  sensor_msgs::msg::NavSatFix nav_sat_fix_origin;
```

Comment:
```suggestion
  sensor_msgs::msg::NavSatFix nav_sat_fix_origin_;
```

Since this is a private member variable, consider refactoring it this way.